### PR TITLE
Update QUEST_LV_0100.tsv

### PR DESCRIPTION
**Pull request by stvkno**
Updated sentences to sound more natural in English

### CODE REVIEW

**Comment by stvkno** [comment]
Sent request without seeing I need to remove untranslated Korean text. How do I do that?

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -9143,15 +9143,15 @@ QUEST_LV_0100_20151001_009138	Fortunately, you were able to restore the sanctum.
 QUEST_LV_0100_20151001_009139	{memo X}Go to the camp of Anastopa
 QUEST_LV_0100_20151001_009140	{memo X}As Damijonas has told you, let's go to the camp of Anastos if there is someone called Albinas in Anastos.
 QUEST_LV_0100_20151001_009141	{memo X}Retrieve the research materials
-QUEST_LV_0100_20151001_009142	{memo X}It seems that you would have to bring important materials to Gedas to obtain some information regarding Albinas. Please find the research materials from the monsters nearby. You would be able to obtain them by defeating them, but it will be helpful when you press Spacebar to check which monsters has many research documents.
+QUEST_LV_0100_20151001_009142	{memo X}It seems that you will have to bring important materials to Gedas to obtain some information regarding Albinas. Please find the research materials from the monsters nearby. You would be able to obtain them by defeating them, but it will be helpful to press space bar to check which monsters has many research documents.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: to press space bar -> to press the space bar key

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -9384,19 +9384,19 @@ QUEST_LV_0100_20151001_009379	Demon Lord Hauberk told you that he is now ready t
...
 QUEST_LV_0100_20151001_009381	Defeat the boss monster
 QUEST_LV_0100_20151001_009382	Hauberk is getting ready to say goodbye.
-QUEST_LV_0100_20151001_009383	Hauberk told you that he needs to keep the promise he made with Agatas so he asked you to hand over the spirit seperating potion to him. If Agatas is still unconcious, put it inside his pocket.
+QUEST_LV_0100_20151001_009383	Hauberk told you that he needs to keep the promise he made with Agatas so he asked you to hand over the spirit separating potion to him. If Agatas is still unconscious, put it inside his pocket.
 QUEST_LV_0100_20151001_009384	Endless deals
 QUEST_LV_0100_20151001_009385	Talk to Squire Williya
 QUEST_LV_0100_20151001_009386	Squire Williya and Pardone Erikas are having disputes at Mochia Forest. Talk to Squire, Williya what's going on.
-QUEST_LV_0100_20151001_009387	Collect Blue paints
-QUEST_LV_0100_20151001_009388	Squire Williya told you taht as a return for handing over the scroll for the deads, Pardoner Erikas wants Blue paints. Defeat the monsters at Apgaudin Three Ways and collect Blue paints.
+QUEST_LV_0100_20151001_009387	Collect blue paint
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv]: blue paint -> Blue Paint

**Comment by Sunaries** [comment]
Good PR. Some minor changes as you can see from my notes. To remove the Korean text, just leave the line blank and it should be fine. :smile:

**Comment by ttgmichael** [comment]
:+1: 

Note: Memo X lines are lines that are not used in the game. You could edit them if it's bothersome to your eyes on Notepad, but it's just redundant work (as most Memo X lines are duplicates) if you do them all :P

**Comment by Sunaries** [comment]
Good to merge. :+1:

**Comment by imcgames** [comment]
:+1:

**Pull request merged**